SudoRules-V20.1-config.clp

```diff
@@ -43,7 +43,7 @@
 
 ;;; define your general CSP-Rules installation directory (including the ending directory symbol / or \)
 ;;; CSP-Rules-V2.1 will be installed inside this general CSP-Rules installation directory
-(defglobal ?*CSP-Rules* = "/Users/berthier/Documents/Projets/CSP-Rules/") ; <<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<
+(defglobal ?*CSP-Rules* = "../") ; <<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<
```

Comment:
Even if this doesn't work when you launch CLIPS from the Finder, the existing default doesn't work on anyone else's machine except your own.  At least this works if the user launches from the command line at the project root.